mail/rspamd/src/opnsense/service/templates/OPNsense/Rspamd/phishing.conf

```diff
@@ -3,7 +3,9 @@
 #
 {% if helpers.exists('OPNsense.Rspamd.general.enabled') and OPNsense.Rspamd.general.enabled == '1' and helpers.exists('OPNsense.Rspamd.phishing') %}
     openphish_enabled = {% if helpers.exists('OPNsense.Rspamd.phishing.openphish_enabled') and OPNsense.Rspamd.phishing.openphish_enabled == '1' %}true{% else %}false{% endif %};
+    {% if helpers.exists('OPNsense.Rspamd.phishing.openphish_map') and OPNsense.Rspamd.phishing.openphish_map != '' %}openphish_map = {{ OPNsense.Rspamd.phishing.openphish_map }}{% endif %};
```

Comment:
is this working because the semicolon is outside of the if - bottom is the same.

Reply:
please also note that the template engine kills the newline if the line ends with a closing tag so you may have to add an additional newline.

Reply:
Thx ... forgot this one 👍